docs/common/rapids/version.py

```diff
@@ -0,0 +1,30 @@
+# =============================================================================
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+import os.path
+import re
+
+
+def read_version():
+    top = os.path.normpath(os.path.join(os.path.dirname(__file__), "../../.."))
+
+    with open(os.path.join(top, "VERSION")) as f:
```

Comment:
In the Python packages, we symlink the VERSION to an easily accessible location. See [the cudf package](https://github.com/rapidsai/cudf/tree/branch-24.02/python/cudf/cudf) and its [_version.py](https://github.com/rapidsai/cudf/blob/branch-24.02/python/cudf/cudf/_version.py). Can we do the same here, by symlinking `VERSION` to `docs/common/VERSION`?

Alternatively, see the suggestion below to get this information directly from `cudf`. That would be even better.

Reply:
I went with the proposal to get this from `cudf.__version__`.